content/docs/components-and-props.md

```diff
@@ -16,23 +16,23 @@ prev: rendering-elements.html
 next: state-and-lifecycle.html
 ---
 
-Components let you split the UI into independent, reusable pieces, and think about each piece in isolation. This page provides an introduction to the idea of components. You can find a [detailed component API reference here](/docs/react-component.html).
+元件使你可以將 UI 拆分成獨立且可復用的程式碼，並且專注於各別程式碼的思考。本章節旨在介紹元件的相關概念，你也可以在此參閱[詳細的 API 文件](/docs/react-component.html)。
 
-Conceptually, components are like JavaScript functions. They accept arbitrary inputs (called "props") and return React elements describing what should appear on the screen.
+概念上來說，元件就像是 JavaScript 的函式，它接收任意的參數（稱之為 "props"）並且回傳描述畫面的 React 元素。
```

Comment:
```
（稱之為 "props"）
```
We might want to avoid using "" since they're English punctuations. We should discuss this soon to keep everything consistent.